pkgs/by-name/op/openrocket/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
```

Comment:
Use `stdenvNoCC` here so that the C compiler is not pulled in as part of the build.

Reply:
Good catch, thanks